test/server/routes/applications.test.js

```diff
@@ -243,7 +243,7 @@ describe('routes/applications.js', () => {
       .then(response => {
         expect(stubLogger).to.have.been.calledOnce;
         expect(stubSingleKey).to.have.been.calledOnce;
-        expect(stubSingleKey).to.have.been.calledWith('mockAppId', 'mockKeyId', 'mockKeyType', 'mockEnv');
+        expect(stubSingleKey).to.have.been.calledWith('mockAppId', 'mockKeyId', 'mockKeyType', 'oKi1z8KY0gXsXu__hy2-YU_JJSdtxOkJ4K5MAE-gOFVzpKt5lvqnFpVeUjhqhVHZ1K8Hkr7M4IYdzJUnOz2hQw', 'mockEnv');
```

Comment:
Where did the token value come from?

Reply:
this is hardcoded in the fake session data the tests use here https://github.com/companieshouse/applications.developer.web.ch.gov.uk/blob/master/test/server/_fakes/mocks/lib/session.js#L76